docs/proposal/references.bib

```diff
@@ -1,7 +1,7 @@
-@misc{sample-reference,
-	title="{Sample Article Title}",
+@misc{wiki-inverse-square,
+	title="{Inverse Square Law}",
 	year="2019",
 	month="",
-	note="Online; accessed September 24, 2019",
-	howpublished="\url{https://fake_url.fake}"
+	note="Online; accessed October 13, 2019",
+	howpublished="\url{https://en.wikipedia.org/wiki/Inverse-square_law}"
```

Comment:
Citing wikipedia? My grade-school librarian would be disappointed :P

Reply:
Lol, it's a lot less citing and more like here is your quick dose of "what is inverse-square law"